boilerplate/app.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This is the starting point of the application.
+ * It defines the logic to start the api-server with provided configurations.
+ */
+
+'use strict';
+
+const elv = require('elv');
+const Glue = require('@hapi/glue');
+const Promise = require('bluebird');
```

Comment:
For every non-trivial import, it'd be good if you added a short comment about what it is or why it's needed. For example, why Bluebird over native promises?

Reply:
okay